drivers/kw2xrf/Kconfig

```diff
@@ -21,6 +21,7 @@ menuconfig MODULE_KW2XRF
     select MODULE_NETDEV
     select MODULE_NETDEV_IEEE802154
     select MODULE_CORE_THREAD_FLAGS
+    select HAVE_BHP_IRQ_HANDLER
```

Comment:
Not yet, right? 😉

Reply:
Well, there is a "BHP Handler" expected to be called by someone (either any of the `bhp_*` modules or a custom based solution).
Maybe I can find another name to make it less confusing

Reply:
ah! sorry, wrong PR :D

Reply:
done